Database-Features/what-to-do-when-i-see-differences-between-database-size-or.md

```diff
@@ -47,18 +43,18 @@ FROM
 ORDER BY
     MEASURE_TIME DESC;
 ```
-Huge phantom percentage (>5%) for a longer periods of time is not normal.
+Huge phantom percentage (>5%) for longer periods of time is not normal.
```

Comment:
I would say here that ups and downs are normal, a consistent upwards trend is not.

Reply:
How about

> Huge phantom percentage (>5%) for longer periods of time and especially consistent upwards trend are not normal.

?

Just "consistent upwards trend" doesn't look measurable enough and I, as a Support agent, will need to be able to explain it in a crystally clear way.